autopayroll-H2Database/workflows/src/test/kotlin/com/autopayroll/FlowTests.kt

```diff
@@ -0,0 +1,35 @@
+package com.autopayroll
+
+import com.autopayroll.flows.RequestFlowResponder
+import net.corda.testing.node.MockNetwork
+import net.corda.testing.node.MockNetworkParameters
+import net.corda.testing.node.TestCordapp
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class FlowTests {
+    private val network = MockNetwork(MockNetworkParameters(cordappsForAllNodes = listOf(
+        TestCordapp.findCordapp("com.autopayroll.contracts"),
+        TestCordapp.findCordapp("com.autopayroll.flows")
+    )))
+    private val a = network.createNode()
+    private val b = network.createNode()
+
+    init {
+        listOf(a, b).forEach {
+            it.registerInitiatedFlow(RequestFlowResponder::class.java)
+        }
+    }
+
+    @Before
+    fun setup() = network.runNetwork()
+
+    @After
+    fun tearDown() = network.stopNodes()
+
+//    @Test
+//    fun `dummy test`() {
+//
+//    }
+}
```

Comment:
Don't check in commented out code. Having a test class that does no tests is confusing as people look at it expecting to see some tests!

Reply:
The original thought behind the dummy tests is that we want to leave them empty for developers to implement their own unit tests and to stimulate their leaning about the sample. But you are also making a valid point here. That said, I will remove the dummy test for sample and write a special corDapp to teach about how to exhaustively test your corDapp.